packages/flutter/lib/src/material_animated_icons/animated_icons_data.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This file serves as the interface between the public and private APIs for
+// animated icons.
+// The AnimatedIcons class is public and is used to specify available icons,
+// while the _AnimatedIconData interface which used to deliver the icon data is
+// kept private.
+
+part of material_animated_icons;
+
+abstract class AnimatedIcons {
+  static const AnimatedIconData arrow_menu = _$arrow_menu;
+  static const AnimatedIconData menu_arrow = _$menu_arrow;
+}
+
+abstract class AnimatedIconData {
+  const AnimatedIconData();
+}
```

Comment:
run `flutter analyze --dartdocs` to get a list of members that need documentation

Reply:
done, I added some comments to silence the analyzer, though I think some of them don't add any information